### PR TITLE
feat: added llama index search and retrieval using an embedded weaviate vector store

### PR DESCRIPTION
**Pull request by ruiciroT**
I have read the CLA Document and I hereby sign the CLA

### CODE REVIEW

File: tutorials/weaviate_llama_index_search_and_retrieval.ipynb [hunk 1]
```diff
@@ -0,0 +1,676 @@
+{
```
Comment (axiomofjoy) [inline tutorials/weaviate_llama_index_search_and_retrieval.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #2.    assert openai_api_key != "", "❌ Please set your OpenAI API key"
</pre><p>Can remove this line</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/1175/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Comment (axiomofjoy) [inline tutorials/weaviate_llama_index_search_and_retrieval.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #1.    url = "http://storage.googleapis.com/arize-assets/xander/milvus-workshop/milvus_dataset.json"
</pre><p>Change you change this link to http://storage.googleapis.com/arize-assets/xander/arize_docs_corpus.json?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/1175/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Comment (axiomofjoy) [inline tutorials/weaviate_llama_index_search_and_retrieval.ipynb:1]: <p>Is there a class method that enables you to initialize <code>WeaviateVectorStore</code> without supplying the client directly? Maybe a <code>from_defaults</code> class method?  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/1175/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Comment (axiomofjoy) [inline tutorials/weaviate_llama_index_search_and_retrieval.ipynb:1]: <p>This is re-downloading the chatbot data from cloud storage and storing it in a storage context that has its own in-memory vector store (not backed by Weaviate). Can you verify that the query engine is using the Weaviate vector and not the in-memory vector store in this case?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/1175/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Comment (axiomofjoy) [inline tutorials/weaviate_llama_index_search_and_retrieval.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #7.        #vector_store = vector_store
</pre><p>This line can be removed.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/1175/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

**Comment by axiomofjoy** [comment]
Ran the notebook, data appears as expected in Phoenix.

**Comment by axiomofjoy** [comment]
@ruiciroT Formatting is failing. Can you follow the instructions [here](https://github.com/Arize-ai/phoenix/blob/main/DEVELOPMENT.md#setting-up-your-macos-development-environment) to set up a Phoenix development environment and run the following commands

- `hatch run notebooks:clean`
- `hatch run type:check`
- `hatch run style:fix`

This should enable your PR to pass the pipeline.

File: tutorials/weaviate_llama_index_search_and_retrieval.ipynb (hunk 1 above)
Comment (ruiciroT) [reply tutorials/weaviate_llama_index_search_and_retrieval.ipynb:1]: <p>Not really as I am assigning to a llama_index WeaviateVectorStore object the weaviate client that was created above.</p>

Comment (ruiciroT) [reply tutorials/weaviate_llama_index_search_and_retrieval.ipynb:1]: <p>Yes it does replace the vector store and uses weaviate </p>

**Comment by axiomofjoy** [comment]
@ruiciroT Looks like formatting is still failing. Did you manage to set up your Phoenix dev env and run those hatch commands? Happy to help.

**Comment by axiomofjoy** [comment]
@ruiciroT Is this PR ready for review?

**Comment by axiomofjoy** [comment]
Closing this PR, assuming it is out-of-date.

**Pull request closed without merging**